src/Raven.Server/Rachis/AppendEntries.cs

```diff
@@ -13,6 +13,7 @@ public class AppendEntries
         public bool ForceElections { get; set; }
         public long TimeAsLeader { get; set; }
         public int SendingThread { get; set; }
+        public int MinClusterCommandVersion { get; set; }
```

Comment:
I will address the command version related issues in the next PR because it is not really related to the certificate replacement command.
See: http://issues.hibernatingrhinos.com/issue/RavenDB-11694